app/schemas/models/level_session.coffee

```diff
@@ -140,6 +140,10 @@ _.extend LevelSessionSchema.properties,
         date: c.date
           description: 'When the submission achieving this score happened.'
         score: {type: 'number'}  # Store 'time', 'damage-taken', etc. as negative numbers so the index works.
+    capstoneStage:
+      type: 'number'
+      title: 'Capstone Stage'
+      description: 'Current capstone stage of the level. A capstone level is not complete until all stages are complete.'
```

Comment:
Is there a way to check from the level session if capstoneStage has been completed?

Reply:
Yes, the preceding `capstoneStage` must have been completed at some point. However, to know if it is _currently_ complete, you would have to compare the current state of all goals against `level.additionalGoals` for the stage you are interested in.

Reply:
I believe we only need to read the saved level session's value, outside the actual play view of capstone levels, so it should be fine to assume that all previous stages have been completed.